src/pages/blog/starlingx-ptp-multi-instance-features.md

```diff
@@ -0,0 +1,126 @@
+---
+templateKey: blog-post
+title: Precision Time Protocol Enhancements StarlingX 7.0

```

Comment:
```suggestion
title: A Deep Dive Into the Precision Time Protocol Configuration Enhancements

```

Reply:
Since we're well into the 9.0 release cycle, I think it is better to decouple the blog post from the release number when some of the new features were introduced.